api/src/pcapi/routes/backoffice/chronicles/blueprint.py

```diff
@@ -148,4 +224,76 @@ def unpublish_chronicle(chronicle_id: int) -> utils.BackofficeResponse:
+    if not FeatureToggle.WIP_ENABLE_CHRONICLES_IN_BO.is_active():
+        raise NotFound()
+
+    form = forms.AttachProductForm()
+    if not form.validate():
+        mark_transaction_as_invalid()
+        flash(utils.build_form_error_msg(form), "warning")
+        redirect(url_for("backoffice_web.chronicles.details", chronicle_id=chronicle_id), code=303)
+
+    chronicle = chronicles_models.Chronicle.query.get_or_404(chronicle_id)
+    product = offers_models.Product.query.filter(offers_models.Product.extraData["ean"].astext == form.ean.data).first()
+
+    if not product:
+        mark_transaction_as_invalid()
+        flash("Aucune œuvre n'à été trouvée pour cet EAN")
+        return redirect(url_for("backoffice_web.chronicles.details", chronicle_id=chronicle_id), code=303)
+
+    chronicle.products.append(product)
+    db.session.add(product)
+    db.session.flush()
+
+    flash(f"{product.name} a été rattaché à la chronique", "success")
+    return redirect(url_for("backoffice_web.chronicles.details", chronicle_id=chronicle_id), code=303)
+
+
+@chronicles_blueprint.route("/<int:chronicle_id>/detach-product/<int:product_id>", methods=["POST"])
+@atomic()
+@permission_required(perm_models.Permissions.MANAGE_CHRONICLE)
+def detach_product(chronicle_id: int, product_id: int) -> utils.BackofficeResponse:
+    if not FeatureToggle.WIP_ENABLE_CHRONICLES_IN_BO.is_active():
+        raise NotFound()
+
+    deleted = chronicles_models.ProductChronicle.query.filter(
+        chronicles_models.ProductChronicle.productId == product_id,
+        chronicles_models.ProductChronicle.chronicleId == chronicle_id,
+    ).delete(synchronize_session=False)
+    db.session.flush()
+
+    if deleted:
+        flash("Le produit à bien été détaché de la chronique", "success")
+    else:
+        mark_transaction_as_invalid()
+        flash("Le produit n'existe pas ou n'était pas attaché à la chronique", "warning")
+
+    return redirect(url_for("backoffice_web.chronicles.details", chronicle_id=chronicle_id), code=303)
+
+
+@chronicles_blueprint.route("/<int:chronicle_id>/comment", methods=["POST"])
+@atomic()
+def comment_chronicle(chronicle_id: int) -> utils.BackofficeResponse:
+    if not FeatureToggle.WIP_ENABLE_CHRONICLES_IN_BO.is_active():
+        raise NotFound()
+
+    form = forms.CommentForm()
+    if not form.validate():
+        mark_transaction_as_invalid()
+        flash(utils.build_form_error_msg(form), "warning")
+        redirect(url_for("backoffice_web.chronicles.details", chronicle_id=chronicle_id), code=303)
+
+    chronicle = chronicles_models.Chronicle.query.get_or_404(chronicle_id)
+
+    history_api.add_action(
+        history_models.ActionType.COMMENT, author=current_user, chronicle=chronicle, comment=form.comment.data
+    )
+    flash("Le commentaire a été enregistré", "success")
+    return redirect(url_for("backoffice_web.chronicles.details", chronicle_id=chronicle_id), code=303)
```

Comment:
Revenons au même onglet, pour voir le nouveau commentaire :
```suggestion
    return redirect(url_for("backoffice_web.chronicles.details", chronicle_id=chronicle_id, active_tab="history"), code=303)
```